pkg/dbmate/mysql.go

```diff
@@ -19,23 +19,38 @@ type MySQLDriver struct {
 }
 
 func normalizeMySQLURL(u *url.URL) string {
-	normalizedURL := *u
-	normalizedURL.Scheme = ""
+	temporaryURL := *u // make a copy of u
```

Comment:
Nitpick, but we don't need to do this if you don't modify the url, i.e., remove the `temporaryURL.RawQuery = query.Encode()` piece and instead store `query.Encode()` in a new variable. I'd vote for using variables instead of constructing the url in this particular case.